velox/functions/prestosql/types/IPPrefixType.cpp

```diff
@@ -78,14 +94,26 @@ class IPPrefixCastOperator : public exec::CastOperator {
   }
 
  private:
+  static void castToIpAddress(
```

Comment:
To cast functions aren't needed and can become a maintenance issue since they are not called.